File: Orion/Components/ImportExport/Interface/SiriusImport/RestContracts/BaseQuoteRiskMsgTypeRisks.vb
Public Class BaseQuoteRiskMsgTypeRisks
    ' From BaseRiskType
    Public Property BranchCode As String
    Public Property DataModelCode As String
    Public Property RiskDescription As String
    Public Property RiskTypeCode As String
    Public Property RunDefaultRules As Boolean
    Public Property ScreenCode As String
    Public Property XMLDataSet As String
    Public Property ProductBuilderDetail As List(Of BaseProductBuilderRiskType)
    Public Property Taxes As List(Of BaseTaxesType)
    ' BaseQuoteRiskMsgTypeRisks additions
    Public Property RatingSections As List(Of BaseRiskRatingSectionType)
    Public Property RiskFolderKey As Integer
    Public Property RiskFolderKeySpecified As Boolean
    Public Property RiskKey As Integer
    Public Property OriginalRiskKey As Integer
    Public Property OriginalRiskKeySpecified As Boolean
    Public Property SAMStagingRiskKey As Integer
End Class

File: SSP.PureInsuranceRestAPIHandler/SSP.PureInsuranceRestAPIHandler/BaseClasses/BaseQuoteRiskMsgTypeRisks.cs
using System.Collections.Generic;

namespace SSP.PureInsuranceRestAPIHandler.BaseClasses
{
    public class BaseQuoteRiskMsgTypeRisks : BaseRiskType
    {
        public System.Collections.Generic.List<BaseRiskRIArrangementType> RIArrangement { get; set; }
        public System.Collections.Generic.List<BaseRiskRatingSectionType> RatingSections { get; set; }
        public int SAMStagingRiskKey { get; set; }
        public int RiskFolderKey { get; set; }
        public int OriginalRiskKey { get; set; }
        public int RiskKey { get; set; }
        public bool RiskFolderKeySpecified { get; set; }
    }
}
